domains/primitives/block-fees/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+//! Inherents for block-fees pallet
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode};
+use domain_runtime_primitives::Balance;
+use sp_inherents::{Error, InherentData, InherentIdentifier, IsFatalError};
+use sp_std::result::Result;
+
+/// Block-fees inherent identifier.
+pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"blockfee";
```

Comment:
```suggestion
pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"blockfee";
```

This might not be an appropriate name if the inherent data is just Balance. No strong opinion though

Reply:
From my understanding, this is just used to identify the inherent data of the `pallet-block-fees` not really related to what type of the inherent data is.